### ISSUE TITLE
Can't resolve '@babel/runtime/core-js/json/stringify'

### ISSUE THREAD

**Issue by howells** [farwayer/mobx-decorators#15]
For no obvious reason, I'm getting an error with mobx-decorators when I try to build my project: partial error below. I've tried removing my node_modules and reinstalling, but the same error persist. Any ideas?

```
./node_modules/mobx-decorators/esm/decorators/save/save.js
Module not found: Error: Can't resolve '@babel/runtime/core-js/json/stringify' in '/Users/danielhowells/Sites/nolii/node_modules/mobx-decorators/esm/decorators/save'
resolve '@babel/runtime/core-js/json/stringify' in '/Users/danielhowells/Sites/nolii/node_modules/mobx-decorators/esm/decorators/save'
  Parsed request is a module
  using description file: /Users/danielhowells/Sites/nolii/node_modules/mobx-decorators/package.json (relative path: ./esm/decorators/save)
    Field 'browser' doesn't contain a valid alias configuration
  after using description file: /Users/danielhowells/Sites/nolii/node_modules/mobx-decorators/package.json (relative path: ./esm/decorators/save)```

**Comment by andrew-luminal** [comment]
+1 also seeing this exact issue.

**Comment by howells** [comment]
@andrew-luminal roll down to `"@babel/runtime": "7.0.0-beta.55",` - the latest version seemed to remove that entire core-js directory

**Comment by farwayer** [comment]
Sorry for delay, fixed in 5.0.2
